sig-node/annual-report-2021.md

```diff
@@ -0,0 +1,67 @@
+    * [Yes](https://github.com/kubernetes/community/tree/master/sig-node#subprojects)
+  * What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping
+  recordings up to date/trends in community members watching recordings?
+    * The SIG operates two recurring meetings.  The Regular SIG Meeting occurs weekly.  It focuses on top-level trends for the SIG, and provides a forum to discuss enhancements, blocked PRs, and release plans.  Each meeting has approximately 15-30 participants depending on the topic.  The meetings are relatively active.  The Chairs host the SIG meeting.  A member of the SIG reports on velocity metrics, and tries to draw attention to overall trends.  The agenda is open for members of the community to discuss an idea or enhancement.  Discussion may ensue for each topic.  SIG members that have participated in the SIG for an extended period will share historical context.  Recordings are up-to-date.  The Weekly CI meeting focuses on improving CI health and improving the number of SIG participants that have sufficient understanding of our test apparatus.  Each meeting has ~7 participants, but participation may improve with new scheduled time.  Each meeting focuses on supporting active triage and review of the project board.  [Recordings are kept up to date with regular SIG meetings](https://www.youtube.com/playlist?list=PL69nYSiGNLP1wJPj5DYWXjiArF-MJ5fNG).
+* How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being
+retired? Are OWNERS.md files up to date in these areas?
+  * The regular SIG meetings provide a forum to discuss sub-projects closely tied to the Kubernetes release.  This includes cri-api, cri-tools, kubelet, and node-api.  A major focus in the SIG this year has been the graduation of existing features.  Other sub-projects get less frequent updates as development velocity has slowed as the project reached sufficient maturity.
+* Same question as above but for working groups.
+  * There is no regular interlock between working groups and the SIG.  SIG members based on interest and/ or desire may attend particular working group meetings.  The SIG did create a dedicated set of meetings to talk through particular areas like topology management for a limited period of time in order to provide room for discussion outside of the regular SIG meeting.  Once consensus was reached, those meetings ceased.  
+* When was your last monthly community-wide update? (provide link to deck and/or recording)
+  * Last presentation was on April 16, 2020:
+    * [Slides](https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit#heading=h.di6sf3cdf3yr)
+
+## Membership
+
+* Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  * For the SIG leads yes.
+  * For subproject owners, likely there are some inactive maintainers, but it’s up to the subproject to prune their OWNERs files.
```

Comment:
Is there a process to ensure that subprojects are pruning their OWNERS files?

Reply:
I have added this as an action item for 2021 to handle this across all related sub-projects.